dataprofiler/profilers/profile_builder.py

```diff
@@ -322,12 +368,25 @@ def clean_data_and_get_base_stats(df_series, sample_size,
         if not len_df:
             return df_series, {
                 "sample_size": 0, "null_count": 0,
-                "null_types": dict(), "sample": []
+                "null_types": dict(), "sample": [],
+                "min_id": None, "max_id": None
             }
 
         # Pandas reads empty values in the csv files as nan
         df_series = df_series.apply(str)
 
+        # Record min and max index values if index is int
+        min_id = None
+        max_id = None
+        if isinstance(df_series.index, pd.RangeIndex) or \
+                all([isinstance(i, int) for i in df_series.index]):
+            min_id = min(df_series.index)
+            max_id = max(df_series.index)
```

Comment:
Couldn't index ranges also be strings?

Reply:
I believe I tested this and a `pd.RangeIndex.start` and `pd.RangeIndex.stop` have to be integers.
https://pandas.pydata.org/docs/reference/api/pandas.RangeIndex.html
confirmed above

Reply:
Index generally can be strings though: https://stackoverflow.com/questions/41536343/i-have-string-index-in-pandas-dataframe-how-can-i-select-by-startswith

Would this be an issue?

Reply:
It is contained in an if statement which requires it to be an int